Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -2202,7 +2202,9 @@ private static void PerformDefaultPluginInitialization()
             #endif
 
             #if UNITY_EDITOR || UNITY_SWITCH || UNITY_STANDALONE || UNITY_WSA
+                #if !UNITY_STANDALONE_LINUX
```

Comment:
Please fold into outer #if. Probably easiest to just replace UNITY_STANDALONE with UNITY_STANDALONE_OSX || UNITY_STANDALONE_WIN.

Reply:
@Rene-Damm  fixed.